app/manifest_name_to_discover_name.py

```diff
@@ -5133,6 +5133,67 @@
+            'files/derivative/sub-M11/sam-pCm11/digital-traces/pC PHPS XFP M11 20XZ 180425_180713_2_NL_N1,2,3.xml': 'files/derivative/sub-M11/sam-pCm11/digital-traces/pC_PHPS_XFP_M11_20XZ_180425_180713_2_NL_N8_9_10.xml',
+            'files/derivative/sub-M11/sam-pCm11/digital-traces/pC PHPS XFP M11 20XZ 180425_180713_2_NL_N4.xml': 'files/derivative/sub-M11/sam-pCm11/digital-traces/pC_PHPS_XFP_M11_20XZ_180425_180713_2_NL_N8_9_10.xml',
+            'files/derivative/sub-M11/sam-pCm11/digital-traces/pC PHPS XFP M11 20XZ 180425_180713_2_NL_N5 connex.xml': 'files/derivative/sub-M11/sam-pCm11/digital-traces/pC_PHPS_XFP_M11_20XZ_180425_180713_2_NL_N8_9_10.xml',
+            'files/derivative/sub-M11/sam-pCm11/digital-traces/pC PHPS XFP M11 20XZ 180425_180713_2_NL_N5.xml': 'files/derivative/sub-M11/sam-pCm11/digital-traces/pC_PHPS_XFP_M11_20XZ_180425_180713_2_NL_N8_9_10.xml',
+            'files/derivative/sub-M11/sam-pCm11/digital-traces/pC PHPS XFP M11 20XZ 180425_180713_2_NL_N8,9,10.xml': 'files/derivative/sub-M11/sam-pCm11/digital-traces/pC_PHPS_XFP_M11_20XZ_180425_180713_2_NL_N8_9_10.xml',
+            'files/derivative/sub-M134/sam-pCm134/digital-traces/pC XFP M134 20XZ 190806_191012_200130D_lx_F G.xml': 'files/derivative/sub-M134/sam-pCm134/digital-traces/pC_XFP_M134_20XZ_190806_191012_200130D_lx.xml',
+            'files/derivative/sub-M134/sam-pCm134/digital-traces/pC XFP M134 20XZ 190806_191012_200130D_lx_F O.xml': 'files/derivative/sub-M134/sam-pCm134/digital-traces/pC_XFP_M134_20XZ_190806_191012_200130D_lx.xml',
+            'files/derivative/sub-M134/sam-pCm134/digital-traces/pC XFP M134 20XZ 190806_191012_200130D_lx.xml': 'files/derivative/sub-M134/sam-pCm134/digital-traces/pC_XFP_M134_20XZ_190806_191012_200130D_lx.xml',
+            'files/derivative/sub-M16/sam-pCm16/digital-traces/pC PHPS XFP M16 20XZ 180425_180524.xml': 'files/derivative/sub-M16/sam-pCm16/digital-traces/pC_PHPS_XFP_M16_20XZ_180425_180524.xml',
+            'files/derivative/sub-M163/sam-pCm163/digital-traces/pCm163_AAV_Z_20x_20200206_S3_F peach.png.xml': 'files/derivative/sub-M163/sam-pCm163/digital-traces/pCm163_AAV_Z_20x_20200206_S3_N12_connex.xml',
+            'files/derivative/sub-M163/sam-pCm163/digital-traces/pCm163_AAV_Z_20x_20200206_S3_N12 connex.xml': 'files/derivative/sub-M163/sam-pCm163/digital-traces/pCm163_AAV_Z_20x_20200206_S3_N12_connex.xml',
+            'files/derivative/sub-M168/digital-traces/pCm168_AAV_Z_20x_191211_S3B_lx_F beige.xml': 'files/derivative/sub-M168/digital-traces/pCm168_AAV_Z_20x_191211_S3B_lx.xml',
+            'files/derivative/sub-M168/digital-traces/pCm168_AAV_Z_20x_191211_S3B_lx_F G.xml': 'files/derivative/sub-M168/digital-traces/pCm168_AAV_Z_20x_191211_S3B_lx.xml',
+            'files/derivative/sub-M168/digital-traces/pCm168_AAV_Z_20x_191211_S3B_lx_IG connex.xml': 'files/derivative/sub-M168/digital-traces/pCm168_AAV_Z_20x_191211_S3B_lx.xml',
```

Comment:
These files are all pointing to the same place, they also seem wrong to me given what is in the dataset.

Reply:
I see. A minor issue with my script. I will double-check this.